draft-specifications/Model.md

```diff
@@ -526,69 +472,56 @@ local peer, the sent and received values will have the same type.
 
 ## Pass Invariant Equality
 
-A type holds equality invariant over passage between OCapN peers if values of that
-type can be passed from a local peer to a remote peer, then returned to the local peer,
-and the value that was sent is equal to the value received.
+OCapN defines a flavor of equality, nominally Equality.
+Two values may be Equal under Equality.
+Values of different OCapN types are not Equal.
+Values of the same OCapN type may be Equal under conditions specific to their
+shared type.
+All OCapN types maintain pass invariant Equality except Promise.
 
-> Equality does not guarantee a unique identity for all equivalent values.
+All OCapN implementations, regardless of language, must model all OCapN types
+such that Equality for the in-language representation of values is consistent
+with the Equality and Pass Invariance of Equality for their corresponding OCapN
+model type.
 
-Any pair of values of different types are not equal or equivalent regardless of
-their values.
+A type holds Equality invariant over passage between OCapN peers because any
+pair of local values that are Equal can be passed to another and the remote
+values will also be Equal.
 
-> Only values of the same type can be equivalent, per the equivalence relation
-> specified for their common type.
+It follows that, for any local value that is sent to a remote peer and then returned,
+the sent and received values will be Equal.
 
-OCapN holds equality invariant for all [Atoms](#atom) and
-[Containers](#container) passed between peers.
+> OCapN Equal values may be locally distinguishable by oother operators
```

Comment:
```suggestion
> OCapN Equal values may be locally distinguishable by other operators
```

Reply:
I do not regret this keyboard layout, because it ameliorates my emacs-claw, but it does have a cost.